src/frontend/context/BehandlingContext.ts

```diff
@@ -97,7 +98,10 @@ const [BehandlingProvider, useBehandling] = createUseContext(() => {
                 history.push(
                     `/fagsak/${fagsak.data.id}/${åpenBehandling.data.behandlingId}/${sideForSteg.href}`
                 );
-                return;
+            } else if (erViPåUlovligSteg(history.location.pathname, sideForSteg) && sideForSteg) {
+                history.push(
+                    `/fagsak/${fagsak.data.id}/${åpenBehandling.data.behandlingId}/${sideForSteg.href}`
```

Comment:
hvis disse to skal pushe akkurat det samme, kan de legges i samme sjekk? 

(erViPåUdefinertFagsakSide(history.location.pathname) || (erViPåUlovligSteg(history.location.pathname, sideForSteg)) && sideForSteg

Reply:
Ja, godt poeng!